src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int numberRandom = new Random().nextInt(100);
+
+    public String getRandomBall() {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/200539443-a52889bb-3c48-4b03-bead-be7b8b977ac9.png)
You return String.